### ISSUE TITLE
Schema check Issue

### ISSUE THREAD

**Issue by petervanvogelpoel-filmbooking** [wolfgangw/digital_cinema_tools_distribution#53]
I'm getting the folowing on almost all DCPs:

Error: AM d363c7a9-6fb9-4863-8a82-4b21a35535a0: Schema check ❌: PROTO-ASDCP-AM-20040311.xsd /dcpstorage001/2015c2f0-c187-4609-d6b9-08dcd6f8de7d/Filmbooking_com_TestDCP5_1_dcp/./ASSETMAP line 7: 7:0: ERROR: Element '{[http://www.digicine.com/PROTO-ASDCP-AM-20040311#}AnnotationText](http://www.digicine.com/PROTO-ASDCP-AM-20040311#%7DAnnotationText)': This element is not expected. Expected is ( {[http://www.digicine.com/PROTO-ASDCP-AM-20040311#}Creator](http://www.digicine.com/PROTO-ASDCP-AM-20040311#%7DCreator) ).

Any advice?
Peter

**Comment by wolfgangw** [comment]
The authoring software might be mixing up the order of elements.
Can you post ```Filmbooking_com_TestDCP5_1_dcp/./ASSETMAP``` to check?

**Comment by petervanvogelpoel-filmbooking** [comment]
See attached.
[ASSETMAP.zip](https://github.com/user-attachments/files/17031041/ASSETMAP.zip)

**Comment by wolfgangw** [comment]
Ja, it's the order of elements. The schema check correctly complains about the ```AnnotationText``` element sitting in the wrong spot. See xsd/PROTO-ASDCP-AM-20040311.xsd for the sequence definition of that group of elements. You could tell the authoring app devs to correct it.

That said, probably noone cares. To the best of my knowledge, playback systems do not check Interop DCP as rigorously as they are required to do with SMPTE DCP. Never heard of failed ingests for those kinds of errors in Interop DCP. ymmv

**Comment by petervanvogelpoel-filmbooking** [comment]
Hi, thanks for your reply. Helpfull. I'll figure out what's wrong.

**Comment by wolfgangw** [comment]
For reference, the related section with ```AnnotationText``` in the wrong spot:

```xml
<Id>urn:uuid:d363c7a9-6fb9-4863-8a82-4b21a35535a0</Id>
<VolumeCount>1</VolumeCount>
<IssueDate>2024-03-12T08:15:31+00:00</IssueDate>
<Issuer>Doremi Labs, Inc.</Issuer>
<AnnotationText>Filmbooking_com_TestDCP5_1_dcp</AnnotationText>
<Creator>Dcp2000</Creator>
```

Correct order would be

```xml
<Id>urn:uuid:d363c7a9-6fb9-4863-8a82-4b21a35535a0</Id>
<AnnotationText>Filmbooking_com_TestDCP5_1_dcp</AnnotationText>
<VolumeCount>1</VolumeCount>
<IssueDate>2024-03-12T08:15:31+00:00</IssueDate>
<Issuer>Doremi Labs, Inc.</Issuer>
<Creator>Dcp2000</Creator>
```

**Comment by petervanvogelpoel-filmbooking** [comment]
Thanks
